src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
```

Comment:
```suggestion
public class Lottery {
```
тут наслідування не зовсім підходить так як в цих класах різні завдання і різні поля. 
Наслідування можна застосувати, коли в тебе зв'язок одн абстракна сутність і її більш конкретний випадок, і у них спільні поля, які ти хочеш отримати від батька в насліднику. Наприклад, Animal -> Cat, Human -> Student, etc